.github/workflows/docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        pip install pdoc3
+        pip install -r requirements.txt
+    - name: Generate documentation
+      run: |
+        pdoc --html --output-dir docs epispot --template-dir pdoc3_templates --force
```

Comment:
- [ ] Where is this being saved to?
 - [ ] Does this override current documentation?
- [ ] Has this workflow been tested?

Reply:
Saved to docs folder. Everything works and has been tested. This overrides any existing find to get up to date docs.